assessment.js

```diff
@@ -1,8 +1,8 @@
 //We'll learn about require later in the course
 var ask = require('./ask.js');
 
-var answer = 'not empty';
+var answer = 0;
 
-while (answer !== '' && answer !== 'SeCrEt') {
+while (answer <= 7 && answer >= 9) {
```

Comment:
This condition will actually never be true. 7 satisfies the first condition but not the second; 9 satisfies the second condition but not the first; 8 satisfies neither condition. What you probably meant was to use an **or** operator (`||`) rather than an **and** operator (`&&`); in that case, the program would keep guessing until someone guessed 8. 

Also, one other thing to consider is that the result value from `ask` will be a String, so you'll need to either convert that to a Number or handle it as a String.